homeassistant/components/calendar/google.py

```diff
@@ -62,7 +64,14 @@ def __init__(self, calendar_service, calendar_id, search=None):
     @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def update(self):
         """Get the latest data."""
-        service = self.calendar_service.get()
+        from httplib2 import ServerNotFoundError
+
+        try:
+            service = self.calendar_service.get()
+        except ServerNotFoundError:
+            _LOGGER.warning("Unable to connect to Google, using cached data")
+            return False
```

Comment:
The return value is not used, so remove it.

Reply:
What about [this code](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/calendar/__init__.py#L138)?

Reply:
...or do you mean that the `not self.data.update()` check is looking for `None` instead of `False`?

Reply:
oh, didn't realize calendar is using their own stuff. kinda makes sense. Ok in that case it's fine.